src/cogs/misc/main.py

```diff
@@ -131,25 +132,34 @@ async def _uptime(self, ctx: commands.Context):
         await ctx.send(embed=em)
 
     @commands.hybrid_command(name="poll", aliases=["ynpoll", "pollstart"])
-    async def _ynpoll(self, ctx: commands.Context, *, question: str):
+    async def _ynpoll(self, ctx: commands.Context, *, args: str):
         """
         Asks question & then adds checkmark & "x" as reactions.
         Thanks to Spoon for original idea
 
         Make embed -> Add buttons, when buttons are clicked, add them to list and grey out
-        after 30 secs announce result
+        after some user defined time in secs announce result
 
         """
-
+        to_wait = 30
+        question = args
+        if args.startswith("-t "):
+            try:
+                s = args.split(" ")[1]
+                to_wait = int(s)    # behaviour note, passing as the only arg `-t 100` will have that be both the time
+                question = " ".join(args.split(" ")[2:])  # and the question
+            except ValueError or IndexError:
+                pass
+        to_wait = max(1, min(to_wait, 3600))
```

Comment:
Hey I like this little trick

Reply:
This little trick is quite useful to quickly put a min and a max on a value